### PR TITLE
Added How to setup stage-file-proxy

### PR DESCRIPTION
**Pull request by laurielim**
In this PR:
- added new guideline in /docs
- updated readme

### CODE REVIEW

File: docs/drupal_stage_file_proxy.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    ``` php
+    $exclude_module_config = [
+        'stage_file_proxy'
+    ];
+    $settings['config_exclude_modules'] = $exclude_module_config;
+
+    ```
+
+4.  Create project specific make file in `tools/make/project`, for example `stage-file-proxy.mk`
```
Comment (back-2-95) [inline docs/drupal_stage_file_proxy.md:34]: Whole 4. can be replace with

Add these lines to .env file:

```
# Enable these modules on local
DRUPAL_ENABLE_MODULES="stage_file_proxy"
```

Comment (laurielim) [reply docs/drupal_stage_file_proxy.md]: Did the change

File: docs/drupal_stage_file_proxy.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    composer require drupal/stage_file_proxy --dev
+    ```
+
+2.  Add configurations for the module to development environment. Modify (or create) `public/sites/default/dev.settings.php` and add
+    ``` php
+    // Stage file proxy origin - no trailing slash.
+    $config['stage_file_proxy.settings']['origin'] = 'https://url-to-production.fi';
+    ```
+
+3.  Exclude module configurations. Modify `public/sites/default/settings.php` and add
```
Comment (back-2-95) [inline docs/drupal_stage_file_proxy.md:24]: "Or if Omen is used, it does this for you"

File: docs/drupal_stage_file_proxy.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Drupal and Stage File Proxy
+
+Stage File Proxy lets you update the database of your local development instance without having to update your files directory. Stage File Proxy transfers each requested file just in time when it is requested. This is especially useful for large sites with huge numbers of files. If you only visit 2 pages and only need 10 remote files to serve those pages, Stage File Proxy will only transfer those 10 files.
```
Comment (back-2-95) [inline docs/drupal_stage_file_proxy.md:3]: "update the database" is not correct. This module basically converts urls on files when links are rendered. Thus it does not work anything which might be added to e.g. templates without using proper Drupal url generation functions.

**Review by back-2-95 — CHANGES_REQUESTED** [review]
See the comments

File: docs/drupal_stage_file_proxy.md (hunk 2 above)
Comment (laurielim) [reply docs/drupal_stage_file_proxy.md:24]: Not sure which part Omen does; #3?

Comment (back-2-95) [reply docs/drupal_stage_file_proxy.md:24]: Omen excludes stage_file_proxy settings from configuration export.

File: docs/drupal_stage_file_proxy.md (hunk 3 above)
Comment (laurielim) [reply docs/drupal_stage_file_proxy.md:3]: I just copied the module description from https://www.drupal.org/project/stage_file_proxy. i could remove that first line if it is problematic.

Comment (back-2-95) [reply docs/drupal_stage_file_proxy.md:3]: Yeah, that 1st line is misleading.

File: docs/drupal_stage_file_proxy.md (hunk 2 above)
Comment (laurielim) [reply docs/drupal_stage_file_proxy.md]: Removed step 3

**Pull request merged**